README.md

```diff
@@ -32,7 +34,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-I thank Microverse for allowing me to use their template.
+-I thank Microverse for allowing me to use their template.
+
+-Thanks to my coding partners and code reviewers for your good advices to improve my code.
 
 ## 📝 License
 
```

Comment:
- [ ] Good job by making most of the changes I required in the previous review.  But as I mentioned in the previous review, kindly make sure you fix the broken link in the **License section** of the README file, **Line 43** (`[MIT](./MIT.md)`) to make it more professional. I think you can fix this by only adding this [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) license file in your root directory and editing it according to your information.

Reply:
Hello Tresor. I don't know how mix my personal information with MIT license. May you explain me again?

Reply:
Hi @robinbenitezmora 
Sure, Just give me 10 min we can do a zoom call together

Reply:
Hi  @robinbenitezmora
Are you still there? So we can join the call